### ISSUE TITLE
SuiteSparse: permissions issue

### ISSUE THREAD

**Issue by bartoldeman** [easybuilders/easybuild-easyblocks#1157]
```
$ find . 2>&1 | grep Permission
find: ‘./GPUQREngine/Demo’: Permission denied
find: ‘./GPUQREngine/Doc’: Permission denied
find: ‘./GPUQREngine/Lib’: Permission denied
find: ‘./GPUQREngine/Include’: Permission denied
find: ‘./GPUQREngine/Source’: Permission denied
find: ‘./SPQR/Matrix’: Permission denied
find: ‘./SPQR/Demo’: Permission denied
find: ‘./SPQR/Tcov’: Permission denied
find: ‘./SPQR/MATLAB’: Permission denied
find: ‘./SPQR/SPQRGPU’: Permission denied
find: ‘./SPQR/Doc’: Permission denied
find: ‘./SPQR/Lib’: Permission denied
find: ‘./SPQR/Include’: Permission denied
find: ‘./SPQR/Source’: Permission denied
find: ‘./SuiteSparse_GPURuntime/Doc’: Permission denied
find: ‘./SuiteSparse_GPURuntime/Lib’: Permission denied
find: ‘./SuiteSparse_GPURuntime/Include’: Permission denied
find: ‘./SuiteSparse_GPURuntime/Source’: Permission denied

$ ls -ld SPQR
drwxr--r-- 11 ebuser ebuser 4096 Mar 30 12:08 SPQR
```
we're missing x permissions on 3 directories. I will check a bit further later.

**Comment by boegel** [comment]
@bartoldeman Any updates on this?

**Comment by boegel** [comment]
fixed in #1238, thanks @bartoldeman!
